hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/InsertIntoHoodieTableCommand.scala

```diff
@@ -243,6 +256,8 @@ object InsertIntoHoodieTableCommand {
         RECORDKEY_FIELD_OPT_KEY.key -> primaryColumns.mkString(","),
         PARTITIONPATH_FIELD_OPT_KEY.key -> partitionFields,
         PAYLOAD_CLASS_OPT_KEY.key -> payloadClassName,
+        ENABLE_ROW_WRITER_OPT_KEY.key -> enableBulkInsert.toString,
```

Comment:
here enableBulkInsert means using ROW_WRITER, should we also introduce another config to control the ROW_WRITER?

Reply:
Currently we already have the ENABLE_ROW_WRITER_OPT_KEY to config the Row write. The enableBulkInsert is only used to control the row write for bulk insert.

Reply:
so here why there is no option for bulk_insert without row_writer? must in row_writer?

Reply:
you can add one, but make the default as true.